src/install/python/python-frameworks/python-agent-scgi.mdx

```diff
@@ -0,0 +1,19 @@
+---
+componentType: appInfo
+optionType: framework
+headingText: Review start-up steps using SCGI
+---
+
+You need to integrate the Python agent with your application so that your app's major functions and web requests are received by the agent. To integrate the Python agent with your app, you can run the newrelic-admin script in front of your usual app startup command.
+
+You can use the Python agent with SCGI in conjunction with [flup](https://pypi.python.org/pypi/flup).
+
+Here's an example of using the admin script:
```

Comment:
This example is a repeat of what the user would see in the server instructions so just remove this.

Reply:
@hmstepanek I removed this sentence and code snippet about the admin script.